reputation/docs/dynamic-threshold.md

```diff
@@ -0,0 +1,78 @@
+# Dynamic Threshold in RePoE
+## RePoE
+RePoE, Reputation Proof of Eligibility, allows us to decide which witnesses are selected 
+to solve the data request. This proof utilizes the influence of the node operator, in terms 
+of its reputation weight, to calculate the odds  to become eligible for a data request in 
+the active reputation system:
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=Prob&space;=&space;(Rep_i&space;/&space;Total&space;Active&space;Rep)&space;*&space;witnesses" target="_blank"><img src="https://latex.codecogs.com/gif.latex?Prob&space;=&space;(Rep_i&space;/&space;Total&space;Active&space;Rep)&space;*&space;witnesses" title="Prob = (Rep_i / Total Active Rep) * witnesses" /></a>
+
+This probability will be translated in a static threshold that provides the maximum value  
+to obtain a valid RePoE. That limit will be compared with the output of a VRF (Verifiable 
+Random Function) provided by the node operator which takes as seed the data request hash 
+and the last CheckpointBeacon of the chain.
+
+## Why do we need a dynamic threshold?
+
+In case of a proportioned reputation system this feature will not be need. But due to how
+our reputation system works, that reputation distribution is unpredictable and could achieve
+some specific distributions which can interrupt the data request resolution.
+
+Example:\
+Address A: 1000 rep\
+Address B: 1000 rep\
+Address C: 1000 rep\
+Address D: 10 rep\
+Address E: 10 rep\
+-----------------------------\
+Total Rep: 3020 rep
+
+In this case, for a data request which requires 4 witnesses the probabilities for the first 
+two will be 1 (currently it is 1.32 but it is not possible to have a probability bigger than 
+1 for a single node), and for the second two wil be 0.0132. So in most of the cases we only 
+obtain 2 valid witnesses, that are not enough to solve the data request.
+
+This problem appears when the probability to mine a data request exceeds 1, because that part 
```

Comment:
```suggestion
This problem appears when the probability at least one witness exceeds 1, because that implies the 
remaining nodes have not enough probability.  In other words, most of the times not enough witnesses will be eligible to resolve the data request.
```